pps_mw_training/pipelines/cloud_base/training_data.py

```diff
@@ -0,0 +1,192 @@
+from pathlib import Path
+from typing import Any, Tuple
+import json
+import random
+import numpy as np  # type: ignore
+import tensorflow as tf  # type: ignore
+import xarray as xr  # type: ignore
+from scipy.ndimage import rotate  # type: ignore
+from pps_mw_training.utils.scaler import Scaler
+from pps_mw_training.utils.calculate_mean_std import get_std_mean
+
+
+def _load_data(
+    data_files: np.ndarray,
+    input_params: str,
+    fill_value_input: float,
+    fill_value_label: float,
+    training_label_name: bytes,
+    training_label_max: float,
+    training_label_min: float,
+) -> list[np.ndarray]:
+    """Load, scale, and filter data."""
+
+    with xr.open_mfdataset(
+        [f.decode("utf-8") for f in data_files],
+        combine="nested",
+        concat_dim="nscene",
+        engine="h5netcdf",
+    ) as all_data:
+        all_data = all_data.sel(
```

Comment:
all data seems to be selected so I do not see the point of this operation and you can  probably remove  all this.

Reply:
removed